examples/language-sdk-instrumentation/golang-push/rideshare-k6/README.md

```diff
@@ -0,0 +1,26 @@
+# Pyroscope k6 integration
+
+This example demonstrates Pyroscope's k6 integration. This example requires k6
+to be installed. To install k6, see: https://grafana.com/docs/k6/latest/set-up/install-k6/.
+
+## Running the example
+
+To run the example, use
+
+```
+docker compose up
+```
+
+This will start all the necessary services, including:
+
+- A Pyroscope server (http://localhost:4040)
+- Grafana instance with Explore Profiles and a Pyroscope data source installed (http://localhost:3000)
+- An nginx load balancer for the Rideshare services (http://localhost:5001)
```

Comment:
As you mentioned, nginx was added to simplify the load test script by providing a single endpoint, should we document this somewhere?

Reply:
Seems reasonable. Applied in https://github.com/grafana/pyroscope/pull/3648/commits/8655891ce837ce2f9f155d7b32cbaf5063ff31c3.